src/baldr/datetime.cc

```diff
@@ -630,6 +630,7 @@ const std::unordered_map<std::string, size_t> tz_new_to_old_id = {
     {"America/Nuuk", 69}, // start renamed timezones 2023c
     {"Europe/Kyiv", 308},
     {"Pacific/Kanton", 358}, // end renamed timezones 2023c
+    {"WET", 310}             // renamed time zones 2024b
```

Comment:
yeah, sadly docs for tz are not super useful lately.. it's not only this one being deprecated, it's also the other equivalents `MET` & `EET`. also, we should remove them from `tz_name_to_id` so those IDs don't show up anymore when `to_index`, mostly for housekeeping, hardly any consequences

Reply:
Ah yup I didn't check for any other aliased time zones, I think the helper function doesn't really spit those out if both time zones already exist?

Reply:
yeah good point, seems we should adapt that a bit so it fails for these cases